app/controllers/chef/bookings_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Chef::BookingsController < ApplicationController
+  before_action :set_user, only: [:show]
```

Comment:
Can you add a comment here so we remember the difference between the chef and user booking controller, please?